packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -269,12 +285,20 @@ export const initializeChugSplash = async (
       })
     ).wait()
 
-    await (
-      await ChugSplashRegistry.connect(multisig).addVersion(
-        ChugSplashManager.address,
-        await getGasPriceOverrides(provider)
-      )
-    ).wait()
+    try {
+      await (
+        await ChugSplashRegistry.connect(multisig).addVersion(
+          ChugSplashManager.address,
+          await getGasPriceOverrides(provider)
+        )
+      ).wait()
+    } catch (e) {
+      // Silence the error if the version is already set.
+      // This error occurs if we attempt to deploy ChugSplash on a network where it's already been deployed.
+      if (!e.message.includes('version already set')) {
+        throw e
+      }
+    }
```

Comment:
nit: We'll lose some ETH every time this call fails on a live network, so, instead, we could remove the try/catch and only call `addVersion` if `registry.managerImplementations(ChugSplashManager.address)` returns `false`